src/test/ui/issues/issue-53712.stderr

```diff
@@ -2,7 +2,7 @@ error[E0609]: no field `0` on type `[{integer}; 5]`
   --> $DIR/issue-53712.rs:5:9
    |
 LL |     arr.0;
-   |         ^ help: attempting to use tuple indexing on an array; try: `arr[0]`
+   |         ^ help: instead of using tuple indexing, use array indexing: `arr[0]`
```

Comment:
I just realized that the suggestion span is slightly incorrect. The suggestion span should cover the entire expression, otherwise when applying the suggestion with `rustfix`, for example, you'd end up with the following code

```
    arr.arr[0];
```

Reply:
Ah, right you are!